### PR TITLE
[fix] Add Kraken Ping

### PR DESCRIPTION
**Pull request by warittornc**
## Implementation details
- Do 1
- Do 2

## Please ensure the following requirements are met before submitting a pull request:

- [ ] The pull request is targeted against the correct target branch
- [ ] The pull request is linked to an issue with appropriate discussion and an accepted design OR is linked to a spec that describes the work.
- [ ] The pull request includes a description of the implementation/work done in detail.
- [ ] The pull request includes any appropriate unit/integration tests
- [ ] You have added a relevant changelog entry to `CHANGELOG.md`
- [ ] You have re-reviewed the files affected by the pull request (e.g. using the `Files changed` tab in the GitHub PR explorer)

### CODE REVIEW

File: bothan-kraken/src/service.rs [hunk 1]
```diff
@@ -169,25 +177,26 @@ async fn handle_reconnect(
...
     // If the keys are empty, subscribe to a single symbol to keep the connection alive
     // TODO: find a better solution
-    let cmd = match keys.is_empty() {
-        true => Command::Subscribe(vec!["XBT/USD".to_string()]),
-        false => Command::Subscribe(keys),
-    };
+    // let cmd = match keys.is_empty() {
+    //     true => Command::Subscribe(vec!["BTC/USD".to_string()]),
+    //     false => Command::Subscribe(keys),
+    // };
```
Comment (RogerKSI) [inline bothan-kraken/src/service.rs:183]: Do we still need this part? if not, please remove.

File: bothan-kraken/src/service/builder.rs [hunk 2]
```diff
@@ -59,7 +58,7 @@ impl KrakenServiceBuilder {
 
         // Subscribe to a single symbol first to keep connection alive
         // TODO: find a better solution
-        let _ = service.get_price_data(&["XBT/USD"]).await;
+        // let _ = service.get_price_data(&["XBT/USD"]).await;
```
Comment (RogerKSI) [inline bothan-kraken/src/service/builder.rs:61]: this one as well.

File: bothan-kraken/examples/kraken_basic.rs [hunk 3]
```diff
@@ -8,10 +8,11 @@ async fn main() {
     init();
 
     let service = KrakenServiceBuilder::default().build().await;
+
     if let Ok(mut service) = service {
+        let data = service.get_price_data(&["BTC/USD", "ETH/USD"]).await;
+        println!("{:?}", data);
         loop {
```
Comment (RogerKSI) [inline bothan-kraken/examples/kraken_basic.rs:15]: should in the loop?

**Review by RogerKSI — APPROVED** [review]

**Review by colmazia — APPROVED** [review]
I may have to do the same for okx

**Pull request merged**